Extension/src/LanguageServer/client.ts

```diff
@@ -1375,6 +1562,18 @@ export class DefaultClient implements Client {
         const settings: any = {
             C_Cpp: {
                 ...cppSettingsScoped,
+                vcFormat: {
+                  //  ...vscode.workspace.getConfiguration("C_Cpp.vcFormat",this.RootUri),
+                    indent: vscode.workspace.getConfiguration("C_Cpp.vcFormat.indent",this.RootUri),
```

Comment:
If you run the "lint" npm script it should give an error about the missing space after the ",".

Reply:
please also delete the commented out code above this line.